librz/include/rz_analysis.h

```diff
@@ -737,6 +738,15 @@ typedef struct rz_analysis_var_t {
 	int argnum;
 } RzAnalysisVar;
 
+typedef struct rz_analysis_var_global_t {
+	char *name; // name of the variable
+	ut64 addr; // address of the global variable
+	RzType *type; // type of the variable
+	RzVector /*<RzAnalysisVarAccess>*/ accesses; // ordered by offset, touch this only through API or expect uaf
+	char *comment;
```

Comment:
Is having comments here really something that makes sense? What is the advantage over putting a regular comment at the same address?

Reply:
@thestr4ng3r it will be shown everywhere the variable accessed. for example.

Reply:
This could be implemented on top of regular comments too.

Reply:
I tend to agree with @thestr4ng3r . This struct should be about one thing and do it well.